src/tasks/RuntimeconfigParser/RuntimeconfigParser.cs

```diff
@@ -0,0 +1,108 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Reflection.Metadata;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+
+public class RuntimeconfigParserTask : Task
+{
+    /// <summary>
+    /// The path to runtimeconfig.json file.
+    /// </summary>
+    [Required]
+    public string runtimeConfigFile { get; set; } = ""!;
```

Comment:
Should this and other properties start with upper case letter to follow the usual naming convention?

Reply:
Fixed. Thanks!